<launch>

	<node pkg="attack_module" type="attack_gps.py" name="attack_gps" output="screen" clear_params="true">
		<!--<param name="publish_rate" value="10" /> -->
		<!--<param name="sleep_duration" value="100" /> -->
		<!--<param name="sensor_attack_type" value="zero_lat_long" /> -->
		<rosparam file="$(find attack_module)/config/attack_gps_param.yaml" command="load" />
	</node>
</launch>  
